docs/Components/alert.css

```diff
@@ -0,0 +1,42 @@
+.bat-alert {
+    border-radius: 5px;
+    padding: 1rem;
+    width: 100%;
+}
+
+.bat-alert-info {
+    background-color: var(--alert-info-bg);
+    color: var(--alert-info);
+}
+
+.bat-alert-success {
+    background-color: var(--alert-success-bg);
+    color: var(--alert-success);
+}
+
+.bat-alert-warning {
+    background-color: var(--alert-warning-bg);
+    color: var(--alert-warning);
+}
+
+
+.bat-alert-error {
+    background-color: var(--alert-error-bg);
+    color: var(--alert-error);
+}
+
+.bat-alert-dismiss {
+    display: flex;
+    justify-content: space-between;
+
```

Comment:
There shouldn't be any empty lines in the code. Also, be consistent in the spacing between two blocks

Reply:
Thank you for your review, I've updated the code.